src/common.speech/DialogServiceAdapter.ts

```diff
@@ -792,6 +792,7 @@ export class DialogServiceAdapter extends ServiceRecognizerBase {
             context: {
                 interactionId: guid
             },
+            messagePayload: this.activityTemplate ? this.activityTemplate : undefined,
```

Comment:
What is the "persistence" story of the template? It is used for as long as is set and it needs to be reset?

Reply:
Correct, this enables the client to "stamp" their activities and have that persist on following turns, until they set/reset it. Though on the topic of properties, maybe this better belongs in our property collection (per conversation w/Ryan, below)